bundles/org.eclipse.swt/Readme.Win32.md

```diff
@@ -51,10 +51,8 @@ You need to install the following on your system:
 
 In the binary git repo, in the windows project, you can build the natives via the build.xml. To do so:  
 
-* In Eclipse, in the binary project org.eclipse.swt.win32.win32.x86_64, right click on build.xml:  
-* Run As -> Ant Build ...
-* On the Targets tab check the build_libraries target  (it should be the only one check for just the natives).
-* On the JRE tab select "Run in the same JRE as the workspace"
+* In Eclipse, open the _Run Configurations_ dialog:  
+* Run the *Maven* launch configuration called `Build-SWT-native-binaries-for-running-platform`
```

Comment:
Since this works for all platform I think it would be best to add this either to the general `Readme.md` in this project or to the `CONTRIBUTING.md` in the repo root. The reference to it could then also be removed from the `Readme.linux.md`.